docs/guides/migrations/guide.rst

```diff
@@ -0,0 +1,1298 @@
+- Type ``edgedb migrate`` again.
+
+The `EdgeDB tutorial <tutorial_>`_ is a good example of a database 
+set up with both a schema migration and a data migration. Setting 
+up a database with schema changes in one file and default data in 
+a second file is a nice way to separate the two operations and maintain 
+high readability at the same time. These two files can be `seen here 
+<tutorial_files_>`_.
+
+Squashing migrations
+====================
+
+Users often end up making frequent changes to their schema because 
+of how effortless it is to do. (And in the next section we will learn 
+about EdgeDB Watch, which is ever more effortless!) This leads to 
+an interesting side effect: lots of ``.edgeql`` files, many of which 
+represent trials and approaches that don't end up making it to the 
+final form for your schema.
+
+Once you are done, you might want to squash the migrations into a 
+single file. This is especially nice if you need to frequently initialize 
+database instances using the same schema, because all migrations are 
+applied when an instance starts up. You can imagine that the output 
+would be pretty long if you had dozens and dozens of migration files 
+to work through:
+
+.. code-block::
+
+    Initializing EdgeDB instance...
+    Applying migrations...
+    Applied m13brvdizqpva6icpcvmsc3fee2yt5j267uba6jugy6iugcbs2djkq 
+    (00001.edgeql)
+    Applied m1aildofb3gvhv3jaa5vjlre4pe26locxevqok4semmlgqwu3xayaa 
+    (00002.edgeql)
+    Applied m1ixxlsdgrlinfijnrbmxdicmpfav33snidudqi7fu4yfhg4nngoza 
+    (00003.edgeql)
+    Applied m1tsi4amrdbcfjypu72duyckrlvvyb46r3wybd7qnbmem4rjvnbcla 
+    (00004.edgeql)
+    ...and so on...
+    Project initialized.
+
+To squash your migrations, just add ``--squash`` after ``edgedb migration 
+create``. Running this command will first display some helpful info 
+to keep in mind before committing to the operation:
+
+.. code-block::
+
+    Current database revision is: 
+    m16ixoukn7ulqdn7tp6lvx2754hviopanufv2lm6wf4x2borgc3g6a
+    While squashing migrations is non-destructive, 
+    it may lead to manual work if done incorrectly.
+
+    Items to check before using --squash:
+    1. Ensure that `./dbschema` dir is comitted
+    2. Ensure that other users of the database have the revision
+    above or can create database from scratch.
+        To check a specific instance, run:
+        edgedb -I <name> migration log --from-db --newest-first --limit 1
+    1. Merge version control branches that contain schema changes
+    if possible.
+
+    Proceed? [y/n]
+
+Then just press ``y`` to squash all of your existing migrations into 
+a single file.
+
+Fixups during a squash
+----------------------
+
+If your schema doesn't match the schema in the database, EdgeDB will 
+prompt you to create a *fixup* file, which can be useful (as the CLI 
+says) to "automate upgrading other instances to a squashed revision". 
+You'll see fixups inside a folder called ``/fixups``. Their file names 
```

Comment:
```suggestion
You'll see fixups inside ``/dbschema/fixups``. Their file names 
```
to avoid implying fixups will appear at `fixups` from the project root